.github/workflows/release.yaml

```diff
@@ -26,32 +26,6 @@ jobs:
             exit 1
           fi
       - name: Create release
-        uses: actions/create-release@v1
+        uses: ncipollo/release-action@v1
```

Comment:
This is a third party GitHub Action. Are you sure you want to use it?

Reply:
It recommended by github:  See "maintained actions" on https://github.com/actions/create-release